src/Psd/PsdInterleave.cpp

```diff
@@ -5,11 +5,25 @@
 #include "PsdInterleave.h"
 
 #include "PsdUnionCast.h"
+
+#if !defined(USE_SSE)
+#if defined(_M_IX86) || defined(_M_X64)
+#define USE_SSE 1

```

Comment:
Please rename to PSD_USE_SSE to make it consistent with the rest of the code.

Reply:
Done.